include/rotatingMHD/run_time_parameters.h

```diff
@@ -313,6 +313,106 @@ Stream& operator<<(Stream &stream, const DimensionlessNumbers &prm);
 
 
 
+/*!
+ * @brief A base clase for the parameters of solvers, which are based
+ * on projection schemes.
+ *
+ */
+struct ProjectionSolverParametersBase
+{
+  /*!
+   * Constructor which sets up the parameters with default values.
+   */
+  ProjectionSolverParametersBase();
+
+  /*!
+   * @brief Constructor which sets up the parameters as specified in the
+   * parameter file with the filename @p parameter_filename.
+   */
+  ProjectionSolverParametersBase(const std::string &parameter_filename);
+
+  /*!
+   * @brief Static method which declares the associated parameter to the
+   * ParameterHandler object @p prm.
+   */
+  static void declare_parameters(ParameterHandler &prm);
+
+  /*!
+   * @brief Method which parses the parameters from the ParameterHandler
+   * object @p prm.
+   */
+  void parse_parameters(ParameterHandler &prm);
+
+  /*!
+   * @brief Method forwarding parameters to a stream object.
+   *
+   * @details This method does not add a `std::endl` to the stream at the end.
+   *
+   */
+  template<typename Stream>
+  friend Stream& operator<<(Stream &stream, const ProjectionSolverParametersBase &prm);
+
+  /*!
+   * @brief Enumerator controlling which weak form of the convective
+   * term is to be implemented
+   */
+  ConvectiveTermWeakForm            convective_term_weak_form;
+
+  /*!
+   * @brief Enumerator controlling which time discretization of the
+   * convective term is to be implemented
+   */
+  ConvectiveTermTimeDiscretization  convective_term_time_discretization;
+
+  /*!
+   * @brief The parameters for the linear solver used in the
+   * diffusion step.
+   */
+  LinearSolverParameters            diffusion_step_solver_parameters;
+
+  /*!
+   * @brief The parameters for the linear solver used in the
+   * diffusion step.
+   */
+  LinearSolverParameters            projection_step_solver_parameters;
+
+  /*!
+   * @brief The parameters for the linear solver used in the
+   * diffusion step.
+   */
+  LinearSolverParameters            correction_step_solver_parameters;
+
+  /*!
+   * @brief The parameters for the linear solver used in the
+   * poisson pre-step.
+   */
+  LinearSolverParameters            zeroth_step_solver_parameters;
```

Comment:
Change `zeroth_step_solver_parameters` to `initialization_step_solver_parameters`